tasks/test.py

```diff
@@ -218,21 +218,33 @@ def lint_releasenote(ctx):
 @task
 def lint_filenames(ctx):
     """
-    Scan files to ensure there are no filenames containing illegal characters
+    Scan files to ensure there are no filenames too long or containing illegal characters
     """
+    files = ctx.run("git ls-files -z", hide=True).stdout.split("\0")
```

Comment:
@derekwbrown Do we need to check all the files from the repo or just the release notes ? The script already catches a few :
```
Checking filenames for illegal characters
Checking filename length
Error: path releasenotes/notes/enabling-collecting-Node-Labels-from-the-Datadog-Cluster-Agent-8b63e92fedbd8668.yaml is too long
Error: path test/kitchen/test/integration/dd-agent-install-script/rspec/dd-agent-installation-script_spec.rb is too long
Exited with code 1
```
and `dd-agent-installation-script_spec.rb` was there for a long time but I guess this might be because we used a defensive value for `prefix_length`

Reply:
I think it's not necessary for all the files, go files are not affected for example. Yes `prefix_length` is defensive approximation.